src/components/Text.re

```diff
@@ -69,6 +69,25 @@ external make:
     ~value: string=?,
     ~children: React.element=?,
     // React Native Web Props
+    ~rel: [@bs.string] [
+            | `alternate
+            | `author
+            | [@bs.as "dns-prefetch"] `dnsPrefetch
+            | `icon
+            | `license
+            | `next
+            | `pingback
+            | `preconnect
+            | `prefetch
+            | `preload
+            | `prerender
+            | `prev
+            | `search
+            | `stylesheet
+          ]
```

Comment:
@jfrolich curious to know where you find this list :) 
I saw some missings (eg: nofollow, noopenner & more...) and some incorrect (eg: preload, stylesheet and some others). I am referring to https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel (2nd column)